src/ScottPlot/Palettes/xgfs25.cs

```diff
@@ -0,0 +1,22 @@
+﻿/* A 25-color pelette based on Anton Tsitsulin's 12-color palette discussed:
+ * http://tsitsul.in/blog/coloropt/
+ * At the time the license file was accessed (2021-09-03) in the github address:
+ * https://github.com/xgfs/coloropt
+ * the original work was released under a MIT License.
```

Comment:
The palettes are now under [CC0](https://creativecommons.org/share-your-work/public-domain/cc0/).

I would still like to credit him, but I think we can put just his name, the link, and the words CC0.

Reply:
I just pointed to the GitHub repo. Since it's `CC0` I don't think we are actually required to write `CC0` 🙃